include/inexor/vulkan-renderer/wrapper/framebuffer.hpp

```diff
@@ -26,6 +26,7 @@ class Framebuffer {
     Framebuffer &operator=(Framebuffer &&) = default;
 
     [[nodiscard]] VkFramebuffer get() const {
+        assert(m_framebuffer);
```

Comment:
It's debatable whether this is actually useful, but either way `cassert` include is missing and i think clang-tidy complains about it not being `m_framebuffer != nullptr`.

Reply:
You're right: A wrapper should never be in an invalid state where the get method must check with `assert`.
I will remove them entirely in every get method.

Reply:
Done.